test/Node/Type.test.js

```diff
@@ -113,34 +114,52 @@ describe('isIntegerFraction', function () {
 describe('isFraction', function () {
   it('2/3 true', function () {
     assert.deepEqual(
-      Node.Type.isFraction(math.parse('2/3')),
+      Node.Helper.isFraction(math.parse('2/3')),
```

Comment:
you repeat using this `assert.deepEqual(Node.Helper.isFraction` thing a lot - you could generalize that out into a test function in this file and only pass in the function string and true/false (a lot of other test files have patterns like this)